doc/extending/xlsx/xlsx_import.md

```diff
@@ -62,7 +62,8 @@ NW = no warning generated for these conversions.
 - Align :
   - some horizontal alignments. We only support left/right/center.
   - vertical alignement
-  - other align options (wrapText, indent, shrinkToFit, ...) (NW)
+  - some wrappingText modes. We only support wrap/overflow. Clip mode from Google Sheets will be converted to overflow mode.
```

Comment:
Mention that (if I undestand correctly), we do support clip mode in in o_spreadsheet, but it's just not supported in xslx files.